continuous-testing/README.md

```diff
@@ -4,24 +4,24 @@ This this guide, we explore best practices for continuous testing - including ex
 
 ## Use Cases
 
-In this section, we provide information about potentially useful *use cases* for continuous testing. The key idea is to provide some more nuanced context for the use cases such that technology (tool) solutions can be shared in context. 
+In this section, we provide information about potentially useful *use cases* for continuous integration. The key idea is to provide some more nuanced context for the use cases such that technology (tool) solutions can be shared in context. 
 
 [See list of use cases](use-cases/README.md)
 
 ## Trade Studies
 
-Here we provide a table of trade offs, feature comparisons, and metrics for a given set of reference architectures mapped to use case tool solutions. Choice of comparison points are solicited directly from community members feedback.
+Here we provide a table of trade offs, feature comparisons, and metrics for a given set of reference architectures mapped to use case tool solutions. Choice of comparison points are solicited directly from community member feedback.
 
 [See list of trade studies](trade-studies/README.md)
 
 ## Reference Architectures
 
-In this section, we provide reference architectures that map to select use case scenarios from the above section. The key idea is to provide an architecture / flowchart that simplifies the understanding of how a particular class of tools to satisfy a use case could be put together to help with continuous testing needs. 
+In this section, we provide reference architectures that map to select use case scenarios from the above section. The key idea is to provide an architecture / flowchart that simplifies the understanding of how a particular class of tools to satisfy a use case could be put together to help with continuous integration needs. 
```

Comment:
Likewise here, use "testing" not "integration".

Reply:
Fixed.